frontend/tailwind.config.js

```diff
@@ -0,0 +1,12 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./index.html","./src/**/*.{html,js,ts,tsx,jsx}"],
+  theme: {
+    extend: {
+    colors: {
+      primarybuttonColor: '#377AE3', 
```

Comment:
I would prefer to name this just `primary`. It's used on more than just buttons.